update-bot/wlmbots/checker_bot.py

```diff
@@ -116,43 +96,59 @@ def generate_config_table(checker_config):
     return text
 
 
+class ResultCollector(object):
```

Comment:
Because the ResultCollector is very checkerbot-specific and calls checkerbot-specific methods. I could rewrite checkerbot as a class to make it more clear and coherent, would that be ok?